### ISSUE TITLE
item:[itemdata](-amount): command does not consume item

### ISSUE THREAD

**Issue by tsnorara** [Zrips/CMI#4392]
**Description of issue:item:[itemdata](-amount): command does not consume item**

---

**First I apologize for the lack of explanation in the previous post.( https://github.com/Zrips/CMI/issues/4389)
What I want to say is that (item:[itemdata](-amount)!) command has a problem.(https://www.zrips.net/cmi/commands/specialized/)
This command is a command that executes a command by consuming certain items in the player's inventory.

I used interactive-commands and this command(item:[itemdata](-amount)!) to make npc to replace five diamonds with one map.
Like this 
item:diamond-5! cmi give [playername] map 1
It was a very simple command and fortunately NPC was properly created, changing five diamonds and one map.
However, the problem occurred when the name of a particular item was changed.
When you use an anvil or change your name through /cmi itemname [name].
When trying to exchange items with such renamed items, only the map was given without missing the items that should have disappeared.
If the item is not lost, the command should not be executed; if the command is executed, the item should be lost. But neither worked properly. Clearly, it is a command to consume items, but when using renamed items, only commands were executed without the item being used. Like a hasitem:[itemdata](-amount)! <(This is a command that is executed without consuming as long as you have an item.)

+If the item has any nbttag, the item will not be used.
So I want this command to change not to recognize the item with nbttag. Or if there is nbtag in the item, it disappears and the command is executed.
![GOMCAM 20201016_1823580030](https://user-images.githubusercontent.com/46946467/96273249-14d2aa80-100a-11eb-9dbd-0a562570f219.png)
![ezgif-7-1ac76fc14c47](https://user-images.githubusercontent.com/46946467/96273278-1bf9b880-100a-11eb-8115-07430326caf3.gif)
These pictures are bugs that occurred when the name 1 was made using an anvil for the item.

If you can't fix it, I want a solution to solve this problem. :) thank you for reading.
**
```
######################
## item:[itemtag](-amount):  command dosen't take item bug ##
######################
```

---

**Cmi Version (using`/cmi version`):** 
8.7.8.3 sqLite
**Server Type (Spigot/Paperspigot/etc):** 
paperspigot
**Server Version (using `/ver`):** 
1.13.2

**Comment by Zrips** [comment]
FYI you can post message under old ticket, no need to create new one. Anyways, closed that one, we can address this one. 
Tested on 1.16.3 and on 1.13.2, everything seams to be working correctly at this point, items are taken away with or without custom name or lore. Have you tested this with a clean server and only CMI installed to rule out any other plugin which could interfere with this process?

**Comment by tsnorara** [comment]
Yes I have already tested at 1.13.2 state with nothing but cmi.But it always didn't work on the renamed item...
But I only tested it on 1.13.2, so I don't know if it works well in the latest version.If it doesn't work because it's version 1.13.2. It will be a little sad. Could you fix the problem of version 1.13.2? :(

**Comment by Zrips** [comment]
That's the thing, i did tested it on 1.13.2, everything seams to be working as it should, items are consumed properly independent if name was modified or not. Something else is going on here. Which exact paperspigot version are you running? And did you tried it on completely clean server?
